infra/build/build_status/update_build_status.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 Google Inc.
+# Copyright 2021 Google Inc.
```

Comment:
nit: You don't need to update copyright headers if you are modifying them.

Reply:
Done.